{
  "comments": [
    {
      "key": {
        "uuid": "3a7a1c58_91453f93",
        "filename": "components/ntp_snippets/remote/remote_suggestions_provider_impl.cc",
        "patchSetId": 1
      },
      "lineNbr": 493,
      "author": {
        "id": 1163944
      },
      "writtenOn": "2017-10-10T13:14:57Z",
      "side": 0,
      "message": "I am confused. In the description you say that previously failed fetches were ignored, but what does this code do then? Also I do not see how the new code treats failed fetched differently.",
      "range": {
        "startLine": 490,
        "startChar": 2,
        "endLine": 493,
        "endChar": 3
      },
      "revId": "5182f264a65af4acd53c9cdaa8ff4f84dd06392b",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "9544e821_25060110",
        "filename": "components/ntp_snippets/remote/remote_suggestions_provider_impl.cc",
        "patchSetId": 1
      },
      "lineNbr": 493,
      "author": {
        "id": 1161814
      },
      "writtenOn": "2017-10-10T14:07:00Z",
      "side": 0,
      "message": "This CL does not change the behavior for RefetchWhileDisplaying. The change is that this correct error treatment is used for the cases where AreArticlesEmpty().",
      "parentUuid": "3a7a1c58_91453f93",
      "range": {
        "startLine": 490,
        "startChar": 2,
        "endLine": 493,
        "endChar": 3
      },
      "revId": "5182f264a65af4acd53c9cdaa8ff4f84dd06392b",
      "serverId": "3ce6091f-6c88-37e8-8c75-72f92ae8dfba",
      "unresolved": false
    }
  ]
}